plugins/woocommerce-admin/client/payments-welcome/style.scss

```diff
@@ -165,6 +178,72 @@
 				border-radius: 3px;
 			}
 		}
+
+		.connect-account__apms {
+			.components-card__header {
+				padding: $gap-large 10px $gap $gap-large;
+				h1 {
+					font-size: 20px;
+					line-height: 28px;
+				}
+			}
+			.components-card__body {
+				padding: 0;
+
+				.woocommerce-list__item {
+					> .woocommerce-list__item-inner {
+						padding: $gap-large;
+						.woocommerce-list__item-title {
+							color: $gray-900;
+							font-weight: 700;
+							font-size: 14px;
+							line-height: 18px;
+						}
+						.woocommerce-list__item-content {
+							color: $gray-700;
+							font-size: 13px;
+							line-height: 16px;
+							a {
+								text-decoration: none;
+							}
+						}
+						.woocommerce-list__item-after {
+							margin-left: $gap-large;
+						}
+						.woocommerce-list__item-before {
+							img {
+								width: 36px;
+							}
+						}
+					}
+				}
+			}
+			.components-card__footer {
+				padding: $gap $gap-large;
+				> a {
+					text-decoration: none;
+				}
+			}
+		}
+		.connect-account__apms-notice {
+			.components-card__body {
+				padding: 0;
+			}
+			.components-notice {
+				margin: 0;
+				padding: 0;
+				background-color: #f0f6fc;
+				.components-notice__content {
+					margin: 0;
+					padding: $gap 20px;
```

Comment:
_Nitpick_: I personally prefer to stick with predefined gaps is there is no reason to use a specific one. Otherwise, we might end with too many different sizes.